### PR TITLE
{bio}[foss/2016b] HyPhy v2.3.13

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar missing dep or wrong `cURL` version?

File: easybuild/easyconfigs/h/HyPhy/HyPhy-2.3.13-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source_urls = ['https://github.com/veg/hyphy/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['29b72574302c98d38a92d5316b9283caa80a54c3e74c5e278c5bc9ae3d3df499']
+
+builddependencies = [('CMake', '3.4.3')]
+
+configopts = '-DINSTALL_PREFIX=%(installdir)s'
+
+dependencies = [
+    ('cURL', '7.40.0', '', True),
```
Comment (akesandgren) [inline easybuild/easyconfigs/h/HyPhy/HyPhy-2.3.13-foss-2016b.eb]: Please use ('cURL', '7.49.1'), which is available from GCCcore/5.4.0

Comment (akesandgren) [reply easybuild/easyconfigs/h/HyPhy/HyPhy-2.3.13-foss-2016b.eb]: Fixed

**Review by akesandgren — APPROVED** [review]

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/54596d812ce05023d3e28fa108ac1d41 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @pescobar!

**Pull request merged**